apis/medusa/v1alpha1/medusa_types.go

```diff
@@ -181,4 +181,10 @@ type MedusaClusterTemplate struct {
 	// Define the liveness probe settings to use for the Medusa containers.
 	// +optional
 	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
+
+	// PurgeBackups toggles if the medusa backups should be purged nightly or not
+	// Defaults to true.
+	// +kubebuilder:default=true
+	// +optional
+	PurgeBackups bool `json:"purgeBackups,omitempty"`
```

Comment:
You should have this set as *bool (and later check and deref). This is because when used from Golang itself, the defaultValue is not inherited sadly (and so it becomes by default false when merging things).

Reply:
Ah. Thanks.  I'll make these changes.